src/Test/StateMachine/Utils.hs

```diff
@@ -125,7 +125,21 @@ iall
   => (forall i. Sing (i :: ix) -> p @@ i -> Bool) -> t p -> Bool
 iall p = ifoldr (\i x ih -> p i x && ih) True
 
+{-# MINIMAL itraverse | ifor #-}
```

Comment:
I think the pragma should be inside the class
https://downloads.haskell.org/~ghc/7.8.1/docs/html/users_guide/pragmas.html#minimal-pragma